pkg/adaptor/hypervisor/aws/ec2.go

```diff
@@ -76,3 +76,42 @@ func NewEC2Client(cloudCfg Config) (*ec2.Client, error) {
 	client := ec2.NewFromConfig(cfg)
 	return client, nil
 }
+
+func RetrieveMissingConfig(cfg *Config) error {
```

Comment:
nit: Does it make sense to name this method as `AddAWSConfigs` ?

Reply:
maybe it does, the question is whether we should mention this configs are retrieved?

Reply:
Yeah you are right. I couldn't think of a better name other than "RetrieveMissingConfig"